api/invite.go

```diff
@@ -94,9 +94,9 @@ func (c *Client) CreateInvite(
 // JoinedInvite is returned after joining an invite.
```

Comment:
This comment was not subject of this PR, however, it is poorly worded, and seeing as this is more of a general purpose PR and there are other requested changes, we might as well fix this before it is forgotten.

```suggestion
// JoinedInvite is returned after joining a guild using an invite.
```